include/render/pixman.h

```diff
@@ -38,10 +39,11 @@ struct wlr_pixman_texture {
 	struct wlr_pixman_renderer *renderer;
 
 	pixman_image_t *image;
-	uint32_t format;
+	const struct wlr_pixel_format_info *format;
+	pixman_format_code_t pixman_format;
```

Comment:
Nit: no need to store the format here, it can be directly accessed with `pixman_image_get_format`